EIPS/eip-3541.md

```diff
@@ -0,0 +1,50 @@
+---
+eip: 3541
+title: Reject new contracts starting with the 0xEF byte
+author: Alex Beregszaszi (@axic), Paweł Bylica (@chfast), Andrei Maiboroda (@gumb0)
+discussions-to: https://ethereum-magicians.org/t/evm-object-format-eof/5727/4
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-16
+---
+
+## Abstract
```

Comment:
```suggestion
## Simple Summary
Rejects new contracts starting with the 0xEF byte.

## Abstract
```

Reply:
The "Simple Summary" is not part of EIP-1. It has been removed over a year ago, see #2186.

Reply:
Hmm, the template still references it and doesn't mention that it is optional.  We should probably resolve that.  I will bring it up with the other editors.

https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md

Reply:
This has been brought up months ago either on the channel and/or some other PR, and the agreement was to just fix the template, but apparently nobody did it 😉

Reply:
I'm willing to go to Draft without the Simple Summary, but this EIP should really be moving into Review almost immediately after draft since you want it included in London and I *believe* the current "policy" (desired operating procedure) is that things shouldn't even be considered/eligible for inclusion until they are in Review.  The simple solution if you are in a hurry (which I think you are) would be to just add the Simple Summary since we have been making everyone include a Simple Summary for a while until we can resolve the inconsistency between EIP-1 and the Template.

Note: The reason I'm such a stickler on these things is because we are hugely understaffed on the EIP editors front, and having simple hard and fast rules that *everyone* has to follow is *way* less time consuming than having fuzzy rules that we make exceptions for and have to defend those exceptions to authors of other EIPs.  The "rules" I have been following are that "EIPs should be formatted like the template outlines" in this case.

Reply:
Here is my contribution to resolving the problem: #3564

Of course if you think the consensus of editors from >1.5 years ago is not valid or if you think the sentiment has changed, then it can't be easily merged.